src/Sulu/Component/Content/Query/ContentQueryBuilderInterface.php

```diff
@@ -19,7 +19,10 @@ interface ContentQueryBuilderInterface
      * @param string $webspaceKey
      * @param string[] $locales
      *
-     * @return string
+     * @return array{
+     *     0: string,
+     *     1: array<string, array<string, string>>
+     * }
```

Comment:
this returns: `return [$sql2, $additionalParams];` while additional params is mostly constructed with `$additionalParams[$locale][][paramName] = [$paramValue]`